strawberryfields/apps/qchem/vibronic.py

```diff
@@ -49,6 +49,9 @@
 function computes two-mode squeezing parameters :math:`t`, from the molecule's temperature, which
 are required by the GBS algorithm to compute vibronic spectra for molecules at finite temperature.
 The :func:`sample` function then takes the computed GBS parameters and generates samples.
+The :func:`~.VibronicTransition` function returns a custom ``sf`` operation for application of the
+Doktorov operator within the conventional :class:`~.Program` interface, allowing for greater freedom
+on the choice of input state and any subsequent quantum operations.
```

Comment:
```suggestion
The :func:`~.VibronicTransition` function is an ``sf`` operation that can be used within the
conventional :class:`~.Program` interface for application of the Doktorov operator, allowing for
greater freedom on the choice of input state and any subsequent quantum operations.
```
Sorry for the second revision :laughing: